tests/test_settings.py

```diff
@@ -13,6 +13,25 @@
 from figures import settings as figures_settings
 
 
+@pytest.mark.parametrize('env_tokens, expected ', [
+        ({'IS_MULTISITE': True}, True),
+        ({'IS_MULTISITE': False}, False),
+    ])
+def test_is_multisite(env_tokens, expected):
+    """
+    Test env_tokens work properly for Figures multisite settings
+    """
+    with mock.patch('figures.settings.env_tokens', env_tokens):
```

Comment:
I'm hesitant to use `override_settings` because it looks like `override_settings` does not support nested keys and at this time: https://github.com/django/django/blob/1.8.18/django/test/utils.py#L221-L223

I'd rather just touch the Figures nested dict within `ENV_TOKENS` than have to make sure that each test that overrides settings has properly crafted `ENV_TOKENS`. Even though "right now" `ENV_TOKENS` in the Figures `devsite/test_settings.py just defines `FIGURES` as its only key, there is no guarantee this will stay, in which case `override_settings` becomes a problem.

At this time, I also want to avoid spending the time to do yet another rewrite of how Figures settings are retrieved from edx-platform. I'm happy to revisit when we work on the Hawthorn upgrade. I think it would really help if I do a sequence diagram that illustrates the Figures app instantiating. 

After digging through the `override_settings` code, it appears there may be more robust avenue by using signals. But top priority is getting Figures working in a multisite environment

Reply:
yeah, can't help with the future